hub/scripts.js

```diff
@@ -415,8 +415,9 @@
    * Page details are updated based on these values.
    */
   function handlePageDetails() {
-    const paths = window.location.pathname.replace('/bench', '')
+    const paths = window.location.pathname.replace('/bench', '').replace('/_draft', '')
```

Comment:
I was just looking over the previous code and was going to say we could keep it simpler, like this. I was even thinking we can use a small Regex, like `window.location.pathname.replace(/\/(bench|\_drafts)/g, '')` to avoid the double `replace` statement. But it works this way as well

Reply:
Thanks @overmyheadandbody 
like this way without repeating the replace.

Reply:
I tried this one but not getting the local nav 
window.location.pathname.replace(/\/(bench|_drafts)/g, '') 
https://localnav-ondrafts--fedpub--elaineskpt.hlx.page/_draft/de/documentcloud/acrobat/hub/how-to/document.html

Reply:
gotcha the `_draft` should be without `/` then it works

Reply:
I just noticed I put in `drafts` (plural), when the folder name is `draft`. So the regular expression would be `window.location.pathname.replace(/\/(bench|\_draft)/g, '')`. This would replace any instance of `/bench` and `/_draft` with `''`.